content/30-use-pah/_index.md

```diff
@@ -7,106 +7,131 @@ weight = 30
+* **Credential Type**: Ansible Galaxy/Automation Hub API Token
+* **Galaxy Server URL**: <the respective community repo URL>
+* **API Token**: <the token>
 
-* In **YAML requirements** upload the  `requirements.yml` file from your local machine.
+Now the access information is configured but not used by Automation Controller. This is done on **Organization** level.
 
-* Click **Save**
+* Open the `default` **Organization** in Automation Controller and click **Edit**
+* Have a look at the **Galaxy Credentials** field. There is only one credential pointing to Ansible Galaxy. This is the default.
+* Remove the credential and add the one pointing to your PAH.
 
-* In the **Remote** overview tab click **Sync** for the `community` remote
-
-Verify the sync of the collections in **Collections** -> **Collections**, switch the repository filter with the dropdown at the top. There should be a lot of content in the `Red Hat Certified` repo and one collection in the `Community` repo. The 'published' filter will not find anything, since we haven't uploaded any collections we created ourselves.
+{{% notice note %}}
+You could add more credentials here, the order of these credentials sets precedence for the sync and lookup of the content.
+{{% /notice %}}
 
 ### Test Private Automation Hub Integration
 
-Now check that your Automation Controller can actually use the content from your PAH:
+Now check that Automation Controller can actually use the content from your PAH by creating a **Job Template** that requires a collection that is not part of the included Execution Environments:
 
 * Create a new **Project** pointing here: `https://github.com/ansible-learnfest/ee-flow.git`
+  * Have a look at the content on GitHub, esp the `collections/requirements.yml` file
 
-  * Have a look at the content, esp the `collections/requirements.yml` file
-
-* Create an inventory of the target machine or update **Workshop Inventory** by removing `ansible-1` from the list of **Hosts**
+* Update the existing **Workshop Inventory**:
+  * remove `ansible-1` from the list of **Hosts**
+  * Disable `node2` and `node3`
 
 * Create a new **Job Template**:
-
   * **Name**: up to you
+  * **Inventory**: The `Workshop Inventory`
+  * **Project**: The one you just created
+  * **Execution Environment**: `Ansible Engine 2.9 execution environment`
+  * **Playbook**: `deploy-container.yml`
+  * Set the right **Credentials** : `Workshop Credential`
+  * Check **Privilege Escalation**
 
-  * **Inventory**: The one you set up with the Playbook before, it should contain one of your AWS instances, or if you updated the `Workshop Inventory`, use this one.
+* Launch the **Template**
 
-  * **Project**: The one you just created
+If you followed the steps above you should **get an error**. Why? Because You pointed Automation Controller to your PAH but the collection required in the requirements.yml is not there. Check the output of the `Source Control Update` job under **Views**->**Jobs**.
```

Comment:
we can not do that: Project sync will fail since controller will parse the collection/requirements.yml but fail to install the listed collection - so the user can't even create the job template